core/node/api_server/src/tx_sender/mod.rs

```diff
@@ -234,6 +244,14 @@ pub struct TxSenderConfig {
     pub validation_computational_gas_limit: u32,
     pub chain_id: L2ChainId,
     pub whitelisted_tokens_for_aa: Vec<Address>,
+    pub timestamp_asserter_params: Option<TimestampAsserterParams>,
+}
+
+#[derive(Debug, Clone)]
+pub struct TimestampAsserterParams {
```

Comment:
Because it won't work in the external node as it's not aware of that crate.

Reply:
It can be re-exported from the API server (the simplest option, probably). Conceptually, there would be no overhead adding a direct dependency for the EN either because it is included in its dependency graph anyway.